tests/rpc.test.js

```diff
@@ -28,102 +29,85 @@ test("Human readable filter with float value", async () => {
-		for (let j = 0; j < testObject.networks.length; j++) {
-			attemptedConnections += 1;
-			let chainValue = undefined;
-			try {
-				const network = testObject.networks[j];
-				const path = testObject.path;
-				let wsUrl = undefined;
-
-				switch (network) {
-					case "polkadot":
-						wsUrl = "wss://rpc.polkadot.io";
-						break;
-					case "kusama":
-						wsUrl = "wss://kusama-rpc.polkadot.io/";
-						break;
-					case "statemine":
-						wsUrl = "wss://statemine-rpc.polkadot.io/";
-						break;
-					case "statemint":
-						wsUrl = "wss://statemint-rpc.polkadot.io/";
-						break;
-					default:
-						fail("Unknown socket url provided, no connection made.");
-
-				}
-
-				const wsProvider = new WsProvider(wsUrl);
-				let api = await ApiPromise.create({ provider: wsProvider });
-
-				// Build API call
-				const pathParameters = path.split(".");
-				pathParameters.forEach(param => {
-					api = api[param];
-				});
-
-				// Process constants and queries based on parameters prefix
-				switch (pathParameters[0]) {
-					case "consts":
-						chainValue = api.toString();
-						break;
-					case "query":
-						chainValue = await api();
-						chainValue = chainValue.toString();
-						break;
-					default:
-						fail(`Unknown path prefix (${pathParameters[0]}) in ${path}`);
-				}
-			} catch (error) {
-				console.log(error);
+// Test all active RPC paths in individual tests
+const paths = [
+	// Add new RPC paths here for testing coverage
+	{ path: 'consts.system.blockHashCount', network: 'kusama' },
+	{ path: 'consts.balances.existentialDeposit', network: 'kusama' },
+	{ path: 'query.staking.validatorCount', network: 'kusama' },
+	{ path: 'consts.staking.maxNominatorRewardedPerValidator', network: 'kusama' },
+	{ path: 'consts.identity.basicDeposit', network: 'kusama' },
+	{ path: 'query.staking.currentEra', network: 'kusama' },
+	{ path: 'consts.phragmenElection.votingBondBase', network: 'kusama' },
+	{ path: 'query.staking.minNominatorBond', network: 'kusama' },
+	{ path: 'consts.staking.maxNominations', network: 'kusama' },
+	{ path: 'consts.democracy.votingPeriod', network: 'kusama' },
+	{ path: 'consts.crowdloan.minContribution', network: 'kusama' },
+	{ path: 'query.nominationPools.minJoinBond', network: 'kusama' },
+	{ path: 'consts.auctions.endingPeriod', network: 'kusama' },
+	{ path: 'consts.democracy.enactmentPeriod', network: 'kusama' },
+	{ path: 'consts.electionProviderMultiPhase.signedMaxSubmissions', network: 'kusama' },
+	{ path: 'consts.multisig.depositBase', network: 'kusama' },
+	{ path: 'consts.proxy.maxProxies', network: 'kusama' },
+	{ path: 'query.nominationPools.minJoinBond', network: 'kusama' },
+	{ path: 'consts.democracy.voteLockingPeriod', network: 'kusama' },
+	{ path: 'consts.treasury.spendPeriod', network: 'kusama' },
+	{ path: 'consts.electionProviderMultiPhase.maxElectingVoters', network: 'kusama' },
+	{ path: 'consts.assets.assetDeposit', network: 'statemint' },
```

Comment:
I scanned all the markdown docs for RPC components and consolidated those results to only include unique items, which resulted in this list.  As new RPC components are added, this list should expand to cover those paths if not already included.